pages/studentAssessmentInstanceHomework/studentAssessmentInstanceHomework.sql

```diff
@@ -37,7 +37,7 @@ WINDOW
 ORDER BY qo.row_order;
 
 -- BLOCK get_group_info
```

Comment:
Is it still necessary to have this query here, or can we use the `getConfigInfo` function instead?

Reply:
Refactor done in [4baf174](https://github.com/PrairieLearn/PrairieLearn/pull/5290/commits/4baf1740683c146a1b89ad57c707d5f3942bc555) (again apologies, this commit should have been split up a bit more)